bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -43,9 +43,21 @@
         mSavedStateHandler = savedStateHandler;
     }
 
+    void clear(@NonNull Object target) {
+        String uuid = mObjectUuidMap.get(target);
+        if (uuid == null) {
+            return;
+        }
+        clearDataFromDisk(uuid);
```

Comment:
@weisers OK so what I did is replaced the 

```
String uuid = mObjectUuidMap.get(target);
```

with 

```
String uuid = mObjectUuidMap.remove(target);
```

so we'll remove the object and get the UUID in the process if it exists .